### PR TITLE
chore: Schematics update for new app structure

### PR DESCRIPTION
**Pull request by marlass**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -133,6 +133,7 @@
     "@types/express": "^4.17.0",
     "@types/fs-extra": "^9.0.7",
     "@types/i18next": "^12.1.0",
+    "@types/inquirer": "^7.3.1",
```
Comment (marlass) [inline package.json]: I might have added it by accident. Need to check if this is needed

File: projects/schematics/package.json [hunk 2]
```diff
@@ -14,11 +14,13 @@
   "scripts": {
     "build": "yarn clean && ../../node_modules/.bin/tsc -p ./tsconfig.schematics.json",
     "clean": "../../node_modules/.bin/rimraf \"src/**/*.js\" \"src/**/*.js.map\" \"src/**/*.d.ts\"",
-    "test": "yarn clean && ../../node_modules/.bin/jest --config ./jest.schematics.config.js"
+    "test": "yarn clean && ../../node_modules/.bin/jest --config ./jest.schematics.config.js",
+    "test:watch": "yarn test --watch"
```
Comment (marlass) [inline projects/schematics/package.json]: Do we need extra command just for a flag that can be passed like this `yarn test --watch`?

File: projects/schematics/src/add-spartacus/store.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import { Rule, SchematicsException, Tree } from '@angular-devkit/schematics';
+import { NGRX_EFFECTS, NGRX_STORE } from '../shared/constants';
+import { addModuleImport } from '../shared/utils/new-module-utils';
+import { createProgram, saveAndFormat } from '../shared/utils/program';
+import { getProjectTsConfigPaths } from '../shared/utils/project-tsconfig-paths';
+
+/** Migration that ensures that we have correct RouterModule.forRoot set */
```
Comment (marlass) [inline projects/schematics/src/add-spartacus/store.ts]:
```suggestion
/** Migration that ensures that we have correct Store modules set */
```

File: projects/schematics/src/add-spartacus/store.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+export function setupStoreModules(project: string): Rule {
+  return (tree: Tree): Tree => {
+    const { buildPaths } = getProjectTsConfigPaths(tree, project);
+
+    if (!buildPaths.length) {
+      throw new SchematicsException(
+        'Could not find any tsconfig file. Cannot set RouterModule.'
```
Comment (marlass) [inline projects/schematics/src/add-spartacus/store.ts]:
```suggestion
        'Could not find any tsconfig file. Cannot set Store modules.'
```

**Review by znikola — DISMISSED** [review]

**Review by znikola — APPROVED** [review]

**Pull request merged**